File: .agents/skills/pagbank-connect/examples/code/connect-info/request.mjs
/**
 * Valida Connect Key via connect/connectInfo.
 * Node 18+ (fetch nativo). Uso: PAGBANK_CONNECT_KEY=... node request.mjs
 */
const CONNECT_KEY = process.env.PAGBANK_CONNECT_KEY ?? (() => { throw new Error('Defina PAGBANK_CONNECT_KEY'); })();
const BASE_URL = 'https://ws.pbintegracoes.com/pspro/v7';

const res = await fetch(`${BASE_URL}/connect/connectInfo`, {
  headers: {
    Authorization: `Bearer ${CONNECT_KEY}`,
    'Content-Type': 'application/json',
    Platform: 'AI',
  },
});

if (!res.ok) {
  console.error(await res.text());
  process.exit(1);
}

console.log(await res.json());

File: .agents/skills/pagbank-connect/examples/code/connect-info/request.py
#!/usr/bin/env python3
"""Valida Connect Key via connect/connectInfo."""
import os
import requests

CONNECT_KEY = os.environ["PAGBANK_CONNECT_KEY"]
BASE_URL = "https://ws.pbintegracoes.com/pspro/v7"

response = requests.get(
    f"{BASE_URL}/connect/connectInfo",
    headers={
        "Authorization": f"Bearer {CONNECT_KEY}",
        "Content-Type": "application/json",
        "Platform": "AI",
    },
    timeout=30,
)
response.raise_for_status()
print(response.json())
